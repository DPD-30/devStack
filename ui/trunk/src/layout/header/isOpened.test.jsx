import React from 'react'
import { renderToString } from 'react-dom/server'


import LayoutHeaderIsOpened from './isOpened.jsx'

jest.unmock('./isOpened.jsx')


test('LayoutHeaderIsOpened matches snapshot', () => {
	const comp = renderToString(<LayoutHeaderIsOpened />)

	expect(comp).toMatchSnapshot()
})
