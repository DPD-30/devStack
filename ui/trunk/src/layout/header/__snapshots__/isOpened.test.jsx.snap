// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutHeaderIsOpened matches snapshot 1`] = `
<div class="header"
     data-reactroot
>
  <div class="container-fluid">
    <div class="row">
      <div class="col-2">
        <img src="caci-logo.svg"
             alt="CACI logo"
             class="logo"
        >
      </div>
      <div class="appName col-9">
        CACI Challenge App
      </div>
      <div class="col-1">
      </div>
    </div>
  </div>
</div>
`;
